FileIO/TetGenInterface.cpp

```diff
@@ -230,8 +228,18 @@ MeshLib::Mesh* TetGenInterface::readTetGenMesh (std::string const& nodes_fname,
 		return nullptr;
 	}
 
+	MeshLib::Properties properties;
+	// remove material vector again if all mat-ids are "0"
```

Comment:
The comment is different from the code; in the code material vector is removed if any of material ids is 0.